User guide: change URL for BRLTTY to brltty.com. re #6728

josephsl opened the pull request:
Hi,

As noted by a recent Liblouis commit: the new address to BRLTTY is brltty.com. Thanks.

josephsl committed:
User guide: change URL for BRLTTY to brltty.com. re #6728

derekriemer commented:
For simple changes, you are welcome to open just a pull request rather 
than both an issue and a

Did you miss this link?
  In order to use this, you need to install [BRLTTY for Windows 
http://brl.thefreecat.org/brltty/].
  You should download and install the latest installer package, which 
will be named, for example,
On 1/13/2017 8:25 AM, Joseph Lee wrote:
>
> Hi,
>
> As noted by a recent Liblouis commit: the new address to BRLTTY is 
> brltty.com. Thanks.
>
> ------------------------------------------------------------------------
>
>
>         You can view, comment on, or merge this pull request online at:
>
> https://github.com/nvaccess/nvda/pull/6729
>
>
>         Commit Summary
>
>   * User guide: change URL for BRLTTY to brltty.com. re #6728
>
>
>         File Changes
>
>   * *M* user_docs/en/userGuide.t2t
>     <https://github.com/nvaccess/nvda/pull/6729/files#diff-0> (2)
>
>
>         Patch Links:
>
>   * https://github.com/nvaccess/nvda/pull/6729.patch
>   * https://github.com/nvaccess/nvda/pull/6729.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6729>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivZ_wHuQdB1AY87e0qtX0NZrjolV8ks5rR5d8gaJpZM4Li_em>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194

josephsl commented:
I tend to follow the old method of issue/PR combo unless instructed. As for the oversight, thanks for catching it.

 

From: derekriemer [mailto:notifications@github.com] 
Sent: Friday, January 13, 2017 12:02 PM
To: nvaccess/nvda <nvda@noreply.github.com>
Cc: Joseph Lee <joseph.lee22590@gmail.com>; Author <author@noreply.github.com>
Subject: Re: [nvaccess/nvda] User guide: change URL for BRLTTY to brltty.com. re #6728 (#6729)

 

For simple changes, you are welcome to open just a pull request rather 
than both an issue and a

Did you miss this link?
In order to use this, you need to install [BRLTTY for Windows 
http://brl.thefreecat.org/brltty/].
You should download and install the latest installer package, which 
will be named, for example,
On 1/13/2017 8:25 AM, Joseph Lee wrote:
>
> Hi,
>
> As noted by a recent Liblouis commit: the new address to BRLTTY is 
> brltty.com. Thanks.
>
> ------------------------------------------------------------------------
>
>
> You can view, comment on, or merge this pull request online at:
>
> https://github.com/nvaccess/nvda/pull/6729
>
>
> Commit Summary
>
> * User guide: change URL for BRLTTY to brltty.com. re #6728
>
>
> File Changes
>
> * *M* user_docs/en/userGuide.t2t
> <https://github.com/nvaccess/nvda/pull/6729/files#diff-0> (2)
>
>
> Patch Links:
>
> * https://github.com/nvaccess/nvda/pull/6729.patch
> * https://github.com/nvaccess/nvda/pull/6729.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/6729>, or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivZ_wHuQdB1AY87e0qtX0NZrjolV8ks5rR5d8gaJpZM4Li_em>.
>

-- 
------------------------------------------------------------------------


Derek Riemer

* Department of computer science, third year undergraduate student.
* Proud user of the NVDA screen reader.
* Open source enthusiast.
* Member of Bridge Cu
* Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194




—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/6729#issuecomment-272533780> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AHgLkL_RgYIy5RWx3IX7c82vMaCLqa9Xks5rR9hBgaJpZM4Li_em> .

josephsl committed:
Corrected download link for BRLTTY for Windows. re #6728

Reviewed by Derek Riemer: be sure to change the address for the download link as well, as brltty.com also hosts download links.
